nbexchange/handlers/feedback.py

```diff
@@ -102,7 +100,8 @@ def get(self):
                 with open(r.location, "r+b") as fp:
                     f["content"] = base64.b64encode(fp.read()).decode("utf-8")
                 f["filename"] = feedback_name
-                f["timestamp"] = r.timestamp.isoformat()
+                # This matches self.timestamp_format
+                f["timestamp"] = r.timestamp.strftime("%Y-%m-%d %H:%M:%S.%f %Z")
```

Comment:
We did, however it broke things.... switching back to this format works, and all the tests pass.

It may be, in a future piece of work, we have either a config variable or some neutral time-format that's transformed in the plugin...